lib/carrierwave/version.rb

```diff
@@ -1,3 +1,3 @@
 module CarrierWave
-  VERSION = "0.11.2"
+  VERSION = "0.11.3"
```

Comment:
Please do not change the version number. I don't have a plan to release this, because it's too old.

Reply:
Gotcha! Reverted 🙂